hack/gencerts.sh

```diff
@@ -0,0 +1,78 @@
+
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
2018?

Also should we use "Copyright 2018 The Kubernetes Authors." to be consistent with copyrights in other files?

Reply:
fixed